### DISCUSSION TITLE
Question: How to retrieve uploaded files in GitHub conversations?

### DISCUSSION THREAD

**Discussion by d-kleine** [community/community#126281] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is there any way to retrieve a file that have been uploaded directly in GitHub conversation about a commit, issue, discussion? 

For example, when writing a post in a GitHub discussion, drag'n'drop a file (e.g. a screenshot) in the "Write" field, this will upload the file to GitHub. If I accidentally closed the tab before the post was published, is there any way to retrieve this uploaded file again?

**Comment by judahpaul16** [comment, marked as answer]
If you accidentally close a tab before publishing a post with an uploaded file on GitHub, there's no way to retrieve the file as it isn't saved unless the post is submitted. However, GitHub uses Camo, their image proxy service, to serve images securely, but it doesn't store or provide access to unpublished images. For more details on how GitHub handles images and the role of Camo, you can visit their documentation on [user content security](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/about-anonymized-urls).

**Reply by d-kleine** [reply]
Okay, thank you!
